content/4.examples/send-payment.draft.md

```diff
@@ -12,4 +12,75 @@ classDiagram
   }
 
   MyPaymentService ..> GevamuFacade
+```
+
+## Program implementation
+
+Sending payment is quite simple operation. It includes only call of `PaymentFlow`. 
+
+But before starting the flow you should provide payment instruction. It should be created according to chosen payment standard.
+
+```kotlin
+class MyPaymentService(private val gevamuFacade: GevamuFacade) {
+    fun sendPaymentViaGevamu() {
+        // Create payment instruction with payment details
+        val paymentInstruction = PaymentInstruction(/* payment details */)
+        
+        // Specify gateway node for payment processing
+        val gateway = Party(/* gateway node data */)
+        
+        // Use GevamuFacade to send payment instruction to gateway node
+        gevamuFacade.sendPayment(paymentInstruction, gateway)
+    }
+}
+
+class GevamuFacade(private val serviceHub: AppServiceHub): SingletonSerializeAsToken() {
+
+  fun sendPayment(
+    paymentInstruction: PaymentInstruction,
+    gateway: Party
+  ): List<StateAndRef<Payment>> {
+    val flowHandle = serviceHub.startFlow(PaymentFlow(paymentInstruction, gateway, UUID.randomUUID()))
+    return flowHandle.returnValue.get()
+  }
+}
+```
+
+To make a transaction as a network participant, describe Payment in one of formats listed in [`PaymentInstructionFormat`](https://gevamu.github.io/corda-payments-sdk/payments-workflows/com.gevamu.corda.flows/-payment-instruction-format/index.html) enum. 
```

Comment:
This sentence can easily continue paragraph before diagram. Let's move it.

Reply:
Did it in new commit